seed/lib/mappings/data/pm-mapping.json

```diff
@@ -224,6 +332,15 @@
     "type": "float",
     "schema": ""
   },
+  {
+    "display_name": "Occupied Floor Area",
```

Comment:
I don't see this exact field reported from ESPM, though I wonder if there's another name for it? Maybe we could scan through the list to see if there's a potential match